ensime-refactor.el

```diff
@@ -244,6 +244,14 @@ Do not asks user about each one if `ensime-refactor-save-with-no-questions' is n
         (forward-list))
       (insert (concat ": " shortname)))))
 
+(add-hook 'diff-mode-hook
```

Comment:
This will apply to all `diff-mode`s, is that ok?

Maybe adding to `ensime-mode-hook` will be better from a lazy loading perspective.